microsetta_interface/implementation.py

```diff
@@ -949,6 +951,30 @@ def post_update_sample(*, account_id=None, source_id=None, sample_id=None):
     return _refresh_state_and_route_to_sink(account_id, source_id)
 
 
+@prerequisite([SOURCE_PREREQS_MET])
+def get_sample_results(*, account_id=None, source_id=None, sample_id=None):
+    has_error, source_output, _ = ApiRequest.get(
+        '/accounts/%s/sources/%s' %
+        (account_id, source_id)
+    )
+    if has_error:
+        return source_output
+
+    has_error, sample_output, _ = ApiRequest.get(
+        '/accounts/%s/sources/%s/samples/%s' %
+        (account_id, source_id, sample_id))
+    if has_error:
+        return sample_output
+
+    return _render_with_defaults('sample_results.jinja2',
+                                 account_id=account_id,
+                                 source_id=source_id,
+                                 sample=sample_output,
+                                 source_name=source_output['source_name'],
+                                 taxonomy=SERVER_CONFIG["taxonomy_resource"],
```

Comment:
I'm not sure I understand why this is a server configuration detail. What happens when shotgun data are available?